org.kde.ark.json

```diff
@@ -105,6 +105,81 @@
                 }
             ]
         },
+        {
+         "name": "gnustep-make",
+         "buildsystem": "simple",
```

Comment:
Have you tried `autotools` here?

Reply:
All of this is copied from [okular](https://github.com/flathub/org.kde.okular/blob/master/org.kde.okular.json#L43) manifest which guarantees this works so I think if there is any improvement to make then it should be done in both (alternatively shared-module may be proposed). I'll prefer not doing that in this PR though.

Reply:
I'm with @Erick555 here, this is relatively complicated so i'd say let's land this, see if we can get it into shared-module and then do improvemetns?